### ISSUE TITLE
Get rid of useless margin in Popup Menu

### ISSUE THREAD

**Issue by elboulangero** [nicklan/pnmixer#122]
This happens only in Gtk3 version.

The Popup Menu nows display a left margin, useless for us, that we can't get rid of. It looks like it's a place holder meant to holds ticks for checkable items. If you look at the latest version of [Parole](http://docs.xfce.org/apps/parole/start) (it's fairly up to date with Gtk3), you will see that their menus are all the same.

This margin appeared when getting rid of "Deprecated" warnings. After rewriting the popup menu ui file properly for Gtk3, it was there. Apparently we can't get rid of it.

**Comment by hasufell** [comment]
Yeah, and what we do isn't particularly "supported" by upstream, since they think icons are bad and are in the process of removing them altogether to worsen user experience.
